package.json

```diff
@@ -16,22 +17,41 @@
     "rfc"
   ],
   "author": "Jakub Roztocil and Lars Schöning",
-  "main": "dist/index",
+  "main": "dist/es5",
```

Comment:
💛 I believe `bower.json` will need to be similarly updated to point to the correct distribution